DeepSpeech.ipynb

```diff
@@ -1360,13 +1311,20 @@
     "                extra_params['options']      = tf.RunOptions(trace_level=tf.RunOptions.FULL_TRACE)\n",
     "                extra_params['run_metadata'] = loss_run_metadata\n",
     "            \n",
-    "            # Compute over the next batch\n",
-    "            _, batch_accuracy, summary_str = session.run([apply_gradient_op, avg_accuracy, merged], \\\n",
-    "                                                         feed_dict_train, \\\n",
-    "                                                         **extra_params)\n",
-    "\n",
+    "            # Compute the batch\n",
+    "            result = session.run(params, feed_dict_train, **extra_params)\n",
```

Comment:
For now this is just not assumed/done. Shouldn't we wait till this is getting an issue?